docs/content/en/docs/migrate/evolution-klt/_index.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Evolution of KLT
+description: Understand the paradigm of KLT, how it evolved from Keptn v1, and whether you should migrate to KLT
+weight: 5
+hidechildren: false # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+The Keptn products provide tool-agnostic ways
```

Comment:
I think "products" is the wrong word here. That said, I don't know the correct work. "Projects" maybe? Because we currently call KLT a "sub project".

Reply:
Agreed.  I reworded it to be "The Keptn project provides..."  I was tempted to say just "Keptn provides..." but that is a problem if we are copyrighting "Keptn" since a copyrighted term can only be used as an adjective.  We'll get lots more eyes on this before it's done...